app/controllers/sitemaps_controller.rb

```diff
@@ -1,11 +1,9 @@
 class SitemapsController < ApplicationController
   respond_to :xml
-  caches_action :show
 
   def show
-    @jobs = Job.active
-    @posts = Post.published
```

Comment:
There is no `Post` constant in this app (anymore?) so the sitemap just doesn't work right now.

Reply:
🤷‍♂️